config/webhook/manifests.yaml

```diff
@@ -64,6 +64,26 @@ webhooks:
     resources:
     - pods
   sideEffects: None
+- admissionReviewVersions:
```

Comment:
Can we get this added to the helm chart as well? https://github.com/aws/amazon-cloudwatch-agent-operator/blob/main/helm/templates/admission-webhooks/operator-webhook.yaml#L84

Reply:
Can make the changes to the helm chart in a separate PR since we'll need to add the workload webhook as well.